luci/luci-app-ffwizard-falter/luasrc/tools/freifunk/assistent/tools.lua

```diff
@@ -4,6 +4,20 @@ local uci = require "luci.model.uci".cursor()
 
 module("luci.tools.freifunk.assistent.tools", package.seeall)
 
+function wifi_get_mesh_modes(device)
+	local modes = {}
+        local phy = string.gsub(device, "radio", "phy")
+	local iwOutput = sys.exec("iw phy "..phy.." info | grep -A1 \"valid interface combination\" | tail -1")
```

Comment:
there is a libiwinfo lib that also supports lua.we could add the call that iwinfo does and get a struct with the information back

Reply:
For some reason the android app wouldn't let me write the comment here.  So I will repeat it and close the change request (it can be made as an update in the future).

If you can provide a link with documentation, then I can take a look at it. I especially like the idea because the manpage for iw states explicitly _not_ to scrape the output